spec/requests/need_help_with_spec.rb

```diff
@@ -0,0 +1,55 @@
+RSpec.describe "need-help-with" do
+  describe "GET /need-help-with" do
+    let(:selected) { ["Feeling unsafe"] }
+
+    context "without session data" do
+      it "shows the form" do
+        visit need_help_with_path
+
+        expect(page.body).to have_content(I18n.t("coronavirus_form.groups.filter_questions.questions.need_help_with.title"))
+        I18n.t("coronavirus_form.groups.filter_questions.questions.need_help_with.options").each do |option|
+          expect(page.body).to have_content(option)
+        end
+      end
+    end
+
+    context "with session data" do
+      before do
+        page.set_rack_session(need_help_with: selected)
+      end
+
+      it "shows the form with prefilled response" do
+        visit need_help_with_path
+
+        expect(page.body).to have_content(I18n.t("coronavirus_form.groups.filter_questions.questions.need_help_with.title"))
+        expect(page.find("input#option_#{selected.first.parameterize.underscore}")).to be_checked
+      end
+    end
+  end
+
+  describe "POST /need-help-with" do
+    let(:selected) { ["Feeling unsafe"] }
```

Comment:
Just gave these a test, I can't get it to submit.

I think it's because there's a gap between the submitted value which would be `feeling_unsafe` and the tests + validations expectations that it would be the Intl Value `Feeling unsafe`